### DISCUSSION TITLE
Issue with update on Nextcloud

### DISCUSSION THREAD

**Discussion by EgonHeuson** [nextcloud/all-in-one#2120] [Questions] [answered]
Hey!
This morning after one week of holiday I came back to work and checked my NextcloudAIO instance. When connecting, I get this error message: 

<img width="703" alt="2023-03-07_13-00-37" src="https://user-images.githubusercontent.com/32910291/223417494-c42c9302-cdda-4023-8340-f4267c21d343.png">

Translation: "Please use the command line update, the automatic update is disabled in config.php."

Not sure to understand te issue here. I checked the nextcloudAIO admin panel and no update seems to be required (I had to run `sudo docker stop nextcloud-aio-apache` through ssh first as the login was still disabled). I stopped all containers and started them again, but no change. 

I have to precise that I have enabled automatic updates every saturday. Therefore this might come from the last update maybe. I didn't do any manual change on the instance for several weeks now.

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/1787

**Reply by EgonHeuson** [reply]
Ahah Simon, always so quick to provide a solution! ☺️ Thanks!! (sorry didn't see the other discussion)
